localflavor/kw/forms.py

```diff
@@ -3,23 +3,29 @@
 """
 from __future__ import unicode_literals
 
-import re
+import textwrap
 from datetime import date
 
 from django.core.validators import EMPTY_VALUES
 from django.forms import ValidationError
-from django.forms.fields import Field
+from django.forms.fields import RegexField
 from django.utils.translation import gettext_lazy as _
 
-id_re = re.compile(r'''^(?P<initial>\d{1})
-                       (?P<yy>\d\d)
-                       (?P<mm>\d\d)
-                       (?P<dd>\d\d)
-                       (?P<mid>\d{4})
-                       (?P<checksum>\d{1})''', re.VERBOSE)
 
+def has_valid_checksum(value):
```

Comment:
Since you're making this a public function, you should rename this to be more meaningful.  `kw_civil_id_number_has_valid_checksum` could work but I'm also open to other suggestions.